src/api/repositories/TutorRepository.ts

```diff
@@ -0,0 +1,36 @@
+import { Tutor } from '../../models/Tutor';
+
+var tutor_entity_database = [
```

Comment:
@Aristevao please change from `var` to `const`.

Reply:
@bigLucas 
I switch `const` to `var` because the value is redeclared in **line 31**, where I changed the strategy to remove a index from this array.

Do you think the other way was better for this case?

Reply:
@Aristevao simple, if you are **re-attributing** a value to the variable **you should use** `let` **instead of** `const` (don't use `var`, always use `let` or `const`). In this case, you can update to `let`.